src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -1380,6 +1380,26 @@ def getWorkloadOverrides(self):
         """
         return self.data.section_('overrides')
 
+    def setWorkloadOverrides(self, overrides):
+        """
+        _getWorkloadOverrides_
+
+        Get the overrides config section
+        of this workload, creates it if it doesn't exist
+        """
+        
+        if overrides:
+            taskIterator = self.taskIterator()
```

Comment:
actually, isn't it returning only the top level task? Is this override then propagated to all tasks or only the top level one?

Reply:
It should iterate all the task not just toplevel.

childTaskIterator is the one iterates only top level.